devices/src/virtio/block.rs

```diff
@@ -597,6 +609,7 @@ impl VirtioDevice for Block {
         queues: Vec<Queue>,
         mut queue_evts: Vec<EventFd>,
     ) -> ActivateResult {
+        println!("activating block device");
```

Comment:
can we make this a log message or remove it, instead of `println!`. In general we should avoid polluting stdout with debug messages.